bin/varnishd/cache/cache.h

```diff
@@ -748,6 +748,12 @@ VSLb_ts_busyobj(struct busyobj *bo, const char *event, vtim_real now)
 /* cache_vcl.c */
 const char *VCL_Name(const struct vcl *);
 
+#define ASSERT_VCL_ACTIVE()					\
+	do {							\
+		assert(vcl_active == NULL ||			\
+		    vcl_active->temp->is_warm);			\
+	} while (0)
```

Comment:
Something for `cache_varnishd.h` instead, but if you are no longer using it in vmod_debug `cache_vcl.h` could maybe work too.